src/providers/PersistRehydrationProvider.tsx

```
@@ -92,10 +104,13 @@ const PersistRehydrationGate = ({
   toRehydrate,
   fallback,
   waitForQueryPersistRestore,
+  onRehydrated,
   children,
 }: PersistRehydrationGateProps) => {
   // rehydration 완료 여부를 확인하기 위한 상태 관리
   const [rehydrationDone, setRehydrationDone] = useState(false);
+  const onRehydratedRef = useRef(onRehydrated);
+  onRehydratedRef.current = onRehydrated;
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/providers/PersistRehydrationProvider.tsx:113:3
  111 |   const [rehydrationDone, setRehydrationDone] = useState(false);
  112 |   const onRehydratedRef = useRef(onRehydrated);
> 113 |   onRehydratedRef.current = onRehydrated;
      |   ^^^^^^^^^^^^^^^^^^^^^^^ Cannot update ref during render
  114 |
  115 |   useEffect(() => {
  116 |     // rehydration 취소 여부 (cleanup 함수에서 사용, 초깃값은 취소되지 않음)

(react-hooks/refs)